frontend/src/services/graphQLMutation.ts

```diff
@@ -106,3 +106,12 @@ export async function graphQLReadNotice(id: string) {
     { id }
   )
 }
+
+export async function graphQLAddNotifications(serviceId: string, name: string, value: boolean) {
+  return await graphQLBasicRequest(
+    ` mutation query($serviceId: String, $name: String!, $value: Any ) {
+        setAttribute(serviceId: $serviceId, name: $name, value: $value)
+     }`,
+    { serviceId, name, value }
+  )
+}
```

Comment:
This looks like maybe it's the same as the `graphQLSetAttributes` method above it - could that one be used? The one above also puts the data under `$remoteit` which I believe is where @penngrove wants attribute data.